client/pages/ListBikes.tsx

```diff
@@ -38,51 +50,77 @@ const ListBikes = () => {
                 return parentArray.includes(x)
             })
         };
-        if (tags){
+        if (tags) {
             setSelectedBikes([...bikeArray].filter((x) => checkSubset(x.tags, tags) === true));
         }
-        else{
+        else {
             setSelectedBikes(bikeArray);        // Reset the bike list because no tags are selected
         }
     };
 
     return (
-        <View style={styles.bikesContainer}>
-            <View style={styles.dropdownWrapper}>
-                <Text style={styles.dropdownLabel}>Filter Search by Tags</Text>
-                <DropDownPicker
-                    maxHeight={300}
-                    style={styles.dropdownInput}
-                    multiple={true}
-                    min={0}
-                    max={3}
-                    open={open}
-                    value={tags}
-                    items={items}
-                    setOpen={setOpen}
-                    setValue={setTags}
-                    setItems={setItems}
-                    mode='BADGE'
-                    onChangeValue={filterBikes}
-                    showBadgeDot={false}
+        <SafeAreaView style={styles.bikesContainer}>
+            <StatusBar
+                backgroundColor='white'
+                barStyle='dark-content'
+            />
+            <View style={styles.searchRow}>
+                <View style={styles.dropdownWrapper}>
+                    <Text style={styles.dropdownLabel}>Filter Search by Tags</Text>
+                    <DropDownPicker
+                        maxHeight={300}
+                        style={styles.dropdownInput}
+                        multiple={true}
+                        min={0}
+                        max={3}
+                        open={open}
+                        value={tags}
+                        items={items}
+                        setOpen={setOpen}
+                        setValue={setTags}
+                        setItems={setItems}
+                        mode='BADGE'
+                        onChangeValue={filterBikes}
+                        showBadgeDot={false}
+                    />
+                </View>
+                <TouchableOpacity
```

Comment:
Let's wrap this in a view so it's centered at the top of the page and not in the same line as the tags filter.